Extensions/ExternalTfs/Src/Tasks/DownloadExternalBuildArtifacts/download.ts

```diff
@@ -149,6 +149,31 @@ async function main(): Promise<void> {
     return promise;
 }
 
+function executeWithRetries(operationName: string, operation: () => Promise<any>, retryCount): Promise<any> {
+    var executePromise = new Promise((resolve, reject) => {
+        executeWithRetriesImplementation(operationName, operation, retryCount, resolve, reject);
+    });
+
+    return executePromise;
+}
+
+function executeWithRetriesImplementation(operationName: string, operation: () => Promise<any>, currentRetryCount, resolve, reject) {
+    operation().then((result) => {
+        resolve(result);
+    }).catch((error) => {
+        if (currentRetryCount <= 0) {
+            tl.error(tl.loc("OperationFailed", operationName, error));
+            reject(error);
+        }
+        else {
+            console.log(tl.loc('RetryingOperation', operationName, currentRetryCount));
+            tl.debug(error);
```

Comment:
good to log this in console as well as tl.Info as well

Reply:
Info level is not supported by taskLib.